cli/installer/windows/ui.wxs

```diff
@@ -75,4 +75,33 @@
                 </Control>
             </Dialog>
         </UI>
-    </Fragment></Wix>
+    </Fragment>
+    <Fragment>
+        <UI>
+            <Dialog Id="ExitDlg" Width="370" Height="270" Title="!(loc.ExitDialog_Title)">
+                <Control Id="Finish" Type="PushButton" X="236" Y="243" Width="56" Height="17" Default="yes" Cancel="yes" Text="!(loc.WixUIFinish)" />
+                <Control Id="Cancel" Type="PushButton" X="304" Y="243" Width="56" Height="17" Disabled="yes" Text="!(loc.WixUICancel)" />
+                <Control Id="Bitmap" Type="Bitmap" X="0" Y="0" Width="370" Height="234" TabSkip="no" Text="!(loc.ExitDialogBitmap)" />
+                <Control Id="Back" Type="PushButton" X="180" Y="243" Width="56" Height="17" Disabled="yes" Text="!(loc.WixUIBack)" />
+                <Control Id="BottomLine" Type="Line" X="0" Y="234" Width="370" Height="0" />
+                <Control Id="Description" Type="Text" X="135" Y="70" Width="220" Height="40" Transparent="yes" NoPrefix="yes" Text="!(loc.ExitDialogDescription)" />
+                <Control Id="Title" Type="Text" X="135" Y="20" Width="220" Height="60" Transparent="yes" NoPrefix="yes" Text="!(loc.ExitDialogTitle)" />
+                <Control Id="UninstallText" Type="Text" X="135" Y="110" Width="220" Height="120" Transparent="yes" NoPrefix="yes" Hidden="yes" Text="The [ProductName] may have downloaded binaries to %USERPROFILE%\.azd\bin and, depending on how azd was used on this machine, may have downloaded binaries to other users' profile directories in their .azd\bin directory. These binaries may need to be removed manually.
+
+To remove such binaries from your home directory, run:
+
+rmdir %USERPROFILE%\.azd\bin.">
+                    <Condition Action="show"><![CDATA[Installed AND REMOVE=ALL]]></Condition>
```

Comment:
@barnson any idea why uninstalling via maintenance mode will show this text in the custom `ExitDlg` (copied from WiX's `ExitDialog` since I wanted this on uninstall) but not with `msiexec /x`? The `Installed` and `REMOVE` properties are set as I knew they would be so I can't imagine why this doesn't show with `msiexec /x`.

Reply:
https://github.com/Azure/azure-dev/issues/1858